### ISSUE TITLE
What is the highest version of civicrm that is using this extension successfully?

### ISSUE THREAD

**Issue by petednz** [CiviCooP/org.civicoop.membershippayment#6]

**Comment by wmortada** [comment]
We are using this on a site that we've just updated to CiviCRM 5.75. It need a civix update but works fine once that is done.

It would be good to get a civix update merged so that it is easier for others to use.

**Comment by wmortada** [comment]
I've added a new PR with the latest civix update: #7. This is the version that we have working on CiviCRM 5.75.
